CMakeLists.txt

```diff
@@ -1,5 +1,8 @@
 cmake_minimum_required(VERSION 3.1)
-project(apriltag VERSION 3.3.0 LANGUAGES C CXX)
+project(apriltag
+        VERSION 3.3.0
+        DESCRIPTION "AprilTag is a visual fiducial system popular for robotics research."
```

Comment:
This is failing for old CMake versions (here on the re-added 16.04 config). Does it make sense to remove the DESCRIPTION attribute?

```
CMake Error: Could not find cmake module file: CMakeDetermineDESCRIPTIONCompiler.cmake
  CMake Error: Error required internal CMake variable not set, cmake may be not be built correctly.
  Missing variable is:
  CMAKE_DESCRIPTION_COMPILER_ENV_VAR
  CMake Error: Error required internal CMake variable not set, cmake may be not be built correctly.
  Missing variable is:
  CMAKE_DESCRIPTION_COMPILER
  CMake Error: Could not find cmake module file: /root/target_ws/build/apriltag/CMakeFiles/3.5.1/CMakeDESCRIPTIONCompiler.cmake
  CMake Error: Could not find cmake module file: CMakeDetermineAprilTag is a visual fiducial system popular for robotics research.Compiler.cmake
  CMake Error: Error required internal CMake variable not set, cmake may be not be built correctly.
```

Reply:
Sure, if the `DESCRIPTION` is the only thing that makes old CMake and Ubuntu fail, then we can remove it and actually just hard-code it in the .pc file.